<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnwrapGUI</class>
 <widget class="QMainWindow" name="UnwrapGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="backgroundBrush">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="foregroundBrush">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="mnuFileOpen"/>
    <addaction name="mnuFileOpenMask"/>
    <addaction name="mnuFileSave"/>
    <addaction name="mnuFileClose"/>
    <addaction name="separator"/>
    <addaction name="mnuFileQuit"/>
   </widget>
   <addaction name="mnuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mnuFileOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="mnuFileSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="mnuFileQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="mnuFileClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="mnuFileOpenMask">
   <property name="text">
    <string>Open &amp;mask</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
